controllers/getCommentsByArticleId.controllers.js

```diff
@@ -0,0 +1,15 @@
+const {
+  selectCommentsByArticleId,
+} = require("../models/selectCommentsByArticleId.models");
+
+exports.getCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  const {sort_by, order} = req.query
```

Comment:
We shouldn't be using any req.query yet. The sort_by and order should be **default parameters**.